install.sh

```diff
@@ -1,5 +1,37 @@
 #!/usr/bin/env bash
 shopt -s expand_aliases
-cat <(curl -s https://raw.githubusercontent.com/imachug/win-sudo/master/installaliases) >> ~/.bashrc && . ~/.bashrc;ws-start-script
-mkdir -pv ~/bin/win-sudo && cd "$_";init-gitrepo-setup && gramf https://github.com/imachug/win-sudo.git && git pull -q base master
-echo "source ~/bin/win-sudo/s/path.sh" | tee -a ws-files null && echo "source ~/bin/win-sudo/s/aliases" >>~/.bashrc && ws-success
+function ws-script-start {
+  echo "Downloading win-sudo..."
+  trap 'echo "Failed to install, sorry :("; exit 1' ERR
+}
+function ws-success {
+  echo Win-sudo successfully installed!
+  exec bash
+  alias fresh="exec bash"
```

Comment:
Shouldn't these 2 lines be swapped?

Reply:
probably, I just moved those over from the other file I removed